<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,12,3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,0,0,0,0,10">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>功能</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbox_function"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>预览色域</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbox_prev_channel"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>曲线色域</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbox_curv_channel"/>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_main">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(165, 165, 165);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="10,0,10,10,0">
      <item>
       <widget class="QLabel" name="label_prev">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(223, 223, 223);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="slider_right">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MyLabel" name="label_curv">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(223, 223, 223);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_hist">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(223, 223, 223);</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_run">
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFILE">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionNewFile"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuEDIT">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actionundo"/>
   </widget>
   <widget class="QMenu" name="menuINFO">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionabout"/>
   </widget>
   <addaction name="menuFILE"/>
   <addaction name="menuEDIT"/>
   <addaction name="menuINFO"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNewFile">
   <property name="text">
    <string>New File</string>
   </property>
  </action>
  <action name="actionundo">
   <property name="text">
    <string>undo</string>
   </property>
  </action>
  <action name="actionabout">
   <property name="text">
    <string>about</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLabel</class>
   <extends>QLabel</extends>
   <header>MyWidget.MyLabel</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
